Modules/_datetimemodule.c

```diff
@@ -2489,7 +2489,6 @@ delta_new(PyTypeObject *type, PyObject *args, PyObject *kw)
         int x_is_odd;
         PyObject *temp;
 
-        whole_us = round(leftover_us);
```

Comment:
For anyone else reading, the way this diff is displayed is a bit misleading - line 2488 is not shown, but the same value is assigned when `whole_us` is declared, then overwritten here.

An alternative would be to delete 2488 and change this into a declaration, or to keep the declaration and assignment lines separate, but I'm indifferent between the three.